src/core/talent.func.js

```diff
@@ -114,6 +139,17 @@ module.exports = class FunctionTalent extends Talent {
             };
 
             return new Rule(new SchemaConstraint(`${this.id}.${func}-in`, eventSchema, DEFAULT_TYPE, VALUE_TYPE_RAW));
-        }));
+        });
+
+        try {
+            const talentRules = this.getRules();
+            talentRules.excludeOn = Object.keys(this.functions).map(func => `${DEFAULT_TYPE}.${this.id}.${func}-in`);
+            functionInputRules.push(talentRules);
+        }
+        catch(err) {
+            // Ignore, if no triggers are given. It offers talent functions only now
```

Comment:
If there is an error in getRules() (implemented by the developer) then it this would be hidden. I propose to check the error and also place in an error message for errors caused by the getRules implementation of the developers.

Reply:
You're right. I will output a warning message.